fiftyone/utils/annotations.py

```diff
@@ -955,11 +955,12 @@ def _format_attributes(backend, attributes):
         if not attr:
             attr = backend.recommend_attr_tool(name, None)
 
-        attr_type = attr.get("type", None)
-        values = attr.get("values", None)
-        default = attr.get("default", None)
-        mutable = attr.get("mutable", None)
-        read_only = attr.get("read_only", None)
+        _attr = attr.copy()
```

Comment:
Should we use `deepcopy` here instead?

Reply:
The current implementation doesn't modify nested values so it doesn't matter. However, agree that `deepcopy` would be more proper.